google/resumable_media/_upload.py

```diff
@@ -452,7 +452,7 @@ def _process_initiate_response(self, response):
         """
         _helpers.require_status_code(
             response,
-            (http_client.OK,),
+            (http_client.OK, http_client.CREATED),
```

Comment:
Hi @lisandromc, thanks again for sending this change. Could you show me an example of when this fails? At the moment a 201 isn't expected when a resumable session is initialized: https://cloud.google.com/storage/docs/performing-resumable-uploads#initiate-session

Reply:
Friendly ping on this question.

Reply:
Hey Frank. I was so busy dealing with other stuff that I overlooked your question, sorry about that.

```
curl --location --request POST 'https://storage.googleapis.com/dam-staging/Ingest/Blink/Testing/Quinta%20Prueba%20de%20Lich/Photo/Palermo/20191005/dolomites.jpg?GoogleAccessId=digitalassetsmanagement%40production-backup-194719.iam.gserviceaccount.com&Expires=1590596284&Signature=kc1xIWCbHeqr100pSOtlA5GpkO1tLoWgU%2FFUyN3QwGQOOGSOwOoO1XCfqBFAdvYywpTi0QdVjIBnB77nq302o0fkzObgpPCUmCq9uWSS2z9ibkiAQdpwWlR9n4XfC1hn25GYumFjZ161ANxC2M5yrjR91jdYtFu2Z4VGFTN%2BhEl7xtYgQ8GHA%2FWSjfCCEfU%2FY0B4p8GMpiLHL3GZtOmBJvqJsuZdTIKb%2FscqctBNX75VLVw5FaWadkj8s0CyIRLM69jy00aPfNO%2BSrfcm4z7PHE6FV8yeJdewoXvJnRDgT0CZDAU7UzgPA4rhqSutcjQT0VkPrN6CHqniEMvhTWaLg%3D%3D' \
> --header 'x-goog-resumable: start' \
> --header 'x-upload-content-type: application/octet-stream' -D -
HTTP/2 201 
content-type: text/plain; charset=utf-8
x-guploader-uploadid: AAANsUm8Q-L1Oh6afXYzfU15-IxWUj4x_JvH8fXvbdCmv-fEdThwSBo4kh3nTyb-RSuyUoelAOLT94NpMy_aZ7YM89k
location: https://storage.googleapis.com/dam-staging/Ingest/Blink/Testing/Quinta%20Prueba%20de%20Lich/Photo/Palermo/20191005/dolomites.jpg?GoogleAccessId=digitalassetsmanagement%40production-backup-194719.iam.gserviceaccount.com&Expires=1590596284&Signature=kc1xIWCbHeqr100pSOtlA5GpkO1tLoWgU%2FFUyN3QwGQOOGSOwOoO1XCfqBFAdvYywpTi0QdVjIBnB77nq302o0fkzObgpPCUmCq9uWSS2z9ibkiAQdpwWlR9n4XfC1hn25GYumFjZ161ANxC2M5yrjR91jdYtFu2Z4VGFTN%2BhEl7xtYgQ8GHA%2FWSjfCCEfU%2FY0B4p8GMpiLHL3GZtOmBJvqJsuZdTIKb%2FscqctBNX75VLVw5FaWadkj8s0CyIRLM69jy00aPfNO%2BSrfcm4z7PHE6FV8yeJdewoXvJnRDgT0CZDAU7UzgPA4rhqSutcjQT0VkPrN6CHqniEMvhTWaLg%3D%3D&upload_id=AAANsUm8Q-L1Oh6afXYzfU15-IxWUj4x_JvH8fXvbdCmv-fEdThwSBo4kh3nTyb-RSuyUoelAOLT94NpMy_aZ7YM89k
content-length: 0
date: Wed, 20 May 2020 16:19:47 GMT
server: UploadServer
alt-svc: h3-27=":443"; ma=2592000,h3-25=":443"; ma=2592000,h3-T050=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q049=":443"; ma=2592000,h3-Q048=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
```

Reply:
Thanks for clarifying that. This change LGTM!